Manejo de Errores (error-handling.mdx)

emilio93 opened the pull request:
Hola! aquí la propuesta de traducción para la página error-handling. Quedo atento a cualquier sugerencia o corrección requerida.

emilio93 committed:
Rev. 1

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -1,40 +1,39 @@
-curious, you can
-[see the source code for these](https://github.com/blitz-js/blitz/blob/canary/packages/core/src/errors.ts).
+Para usar, importa desde `blitz` y usa como cualquier Error de JavaScript. Si
+eres una persona curiosa, puedes [ver el código fuente de estos](https://github.com/blitz-js/blitz/blob/canary/packages/core/src/errors.ts).
```
Cambia "usar", "importa", y "usa" por "usarlos", "impórtalos" y "úsalos" etc.

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -43,25 +42,24 @@ try {
 } catch (error) {
   if (error.name === "AuthenticationError") {
-    // Handle this error appropriately, like show a login screen
+    // Manejar el error adecuadamente, por ejemplo mostrar una pantalla de login
```
```suggestion
    // Maneja el error adecuadamente. Por ejemplo, muestra una pantalla de login
```

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -106,33 +104,33 @@ function RootErrorFallback({ error, resetErrorBoundary }) {
-local ErrorBoundary. If an error is caught by an `<ErrorBoundary>`
-somewhere down inside your app tree, then it will not reach the root
-ErrorBoundary unless you re-throw it.
+Esto significa que todos los errores seran capturados al menos al nivel raíz.
```
"seran" --> "serán"

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -106,33 +104,33 @@ function RootErrorFallback({ error, resetErrorBoundary }) {
+Esto significa que todos los errores seran capturados al menos al nivel raíz.
+Sin embargo, también puedes agregar un `<ErrorBoundary>` en cualquer lugar de tu
+aplicación y manejar los errores de una manera más localizada. Para hacer esto,
+declara un `useQueryErrorResetBoundary` separado en tu componente y pasalo junto
```
"pasalo" --> "pásalo"

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -106,33 +104,33 @@ function RootErrorFallback({ error, resetErrorBoundary }) {
+la raíz, a menos que se lanze nuevamente.
 
-### Handling Server Errors on the Client {#handling-server-errors-on-the-client}
+### Manejando Errores del Servidor en el Cliente {#handling-server-errors-on-the-client}
```
Manejo de Errores (?)
¿Qué te suena mejora?

JuanM04 reviewed (changes requested):
Increíble, gracias por tu aporte. Aquí te dejo un par de comentarios

emilio93 committed:
Rev. 2

emilio93 committed:
Rev. 3

Fix spelling mistakes

emilio93 committed:
Rev. 4

Other changes. Intents are to use more inclusive language, make the translation look less 'translated'.

emilio93 commented:
Con gusto.

Muchas gracias por las correcciones!

Agregué 3 commits:
271b4156034c995d399364a03f1042faf780c5ea Corrige los errores mencionados.
5ccb245dcaff75d35092dc4344c04c9ea09cff6c Corrige otros errores ortográficos.
bb27e0b4835581d95c8ac05344301b5b32be9e8d Otros cambios de contenido para que el texto parezca menos traducido, además una corrección para lenguaje más inclusivo.

Cualquier otro detalle quedo atento para corregirlo/cambiarlo.

Saludos!

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -1,40 +1,40 @@
 
-Blitz comes with a number of useful errors you can use throughout your
-application.
+Blitz viene con una cantidad de errores útiles que puedes usar en toda tu
```
Acá podés ponder "Blitz viene con errores útiles" y listo, así se entiende mejor

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -43,25 +43,24 @@ try {
 
-You handle errors on the client by using
-[`<ErrorBoundary>`](./error-boundary).
+Para manejar los errores en el cliente, usas un [`<ErrorBoundary>`](./error-boundary).
```
Aquí puede ser "Puedes manejar ... usando un"

JuanM04 commented on app/pages/docs/error-handling.mdx:
```diff
@@ -106,33 +105,33 @@ function RootErrorFallback({ error, resetErrorBoundary }) {
-For errors other than what Blitz provides, it's recommended to create
-custom Error classes. You can then add custom data attributes that help
-you handle the error.
+Para errores adicionales a los que Blitz provee, se recomienda crear clases de
```
"Para **usar** errores"

JuanM04 reviewed (changes requested):
Aquí hay un par de cambios más, y creo que ya estaría para el merge

emilio93 committed:
Rev. 5

emilio93 commented:
Listo!

JuanM04 reviewed (approved):
Listo, buen trabajo!
Si querés, podés empezar la traducción de otra página o revisar una de las PR ya existentes
